TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -1,63 +1,87 @@
-// Note: The correct value for the metric must always reside in indice 1 (second element) of the regex output array
+// Note: The correct value for the metric must always reside in indice 1 (capture group) of the regex output array
+// use outerRegex when there are any warmup/cold runs before measure run that you do not wish to include.
+// if not familiar with regex look at other benchmarks & use https://regex101.com/ :)
 
+/* Format of each Benchmark
+ * 	Benchmarkname: {
+ * 	outerRegex:  Regex with capture group: all text after warmup runs is fine
+ * 
+ *  metrics:
+ *  [  // array of metrics
+ * 		{ //individual metric
+ * 			name: name of the metric
+ *  		regex: Regex with capture group: value for the specific metric
+ *  	}, ...
+ *  
+ *  ]
+ */
 const BenchmarkMetricRegex = {
-    LibertyDayTrader3: {
-        metrics: [
-            {
-                name: "Throughput",
-                regex: /<run runNo="5" runType="measure">[\s\S]*?<metric type="throughput">\n?[\s\S]*?([0-9]*[.]?[0-9]+)<\/data>/,
-                regexRepeat: false
-            }
-        ]
-    },
-    LibertyStartup: {
-        metrics: [
-            {
-                name: "Footprint in kb",
-                outerRegex: /\sWarm\srun\s0\s([\s\S]*)/,
-                regex: /\sFootprint\s\(kb\)=(.*)\s/g,
-                regexRepeat: true
-            },
-            {
-                name: "Startup time in ms",
-                outerRegex: /\sWarm\srun\s0\s([\s\S]*)/,
-                regex: /\sStartup\stime:\s(.*)\s/g,
-                regexRepeat: true
-            }
-        ]
-    },
-    ILOG_WODM: {
-        metrics: [
-            {
-                name: "Global Throughput",
-                regex: /Global Throughput.*?=\s?([0-9]*[.]?[0-9]+)/,
-                regexRepeat: false
-            }
-        ]
-    },
-    SPECjbb2015: {
-        metrics: [
-            {
-                name: "max_jOPS",
-                regex: /RUN RESULT:[\s\S]*?max-jOPS\s?=\s?([0-9]*[.]?[0-9]+)[\s\S]*?critical-jOPS\s?=\s?[0-9]*[.]?[0-9]+[\s\S]*?\n/,
-                regexRepeat: false
-            },
-            {
-                name: "critical_jOPS",
-                regex: /RUN RESULT:[\s\S]*?max-jOPS\s?=\s?[0-9]*[.]?[0-9]+[\s\S]*?critical-jOPS\s?=\s?([0-9]*[.]?[0-9]+)[\s\S]*?\n/,
-                regexRepeat: false
-            }
-        ]
-    },
-    AcmeAirNodejs: {
-        metrics: [
-            {
-                name: "Throughput",
-                regex: /Throughput: (\d*\.?\d*)/, //Example: "Throughput: 3554.1"
-                regexRepeat: false
-            }
-        ]
-    }
+
+	LibertyDayTrader3 : 
```

Comment:
This should be `LibertyDayTrader` without the 3 since that's what's expected in BenchmarkMetricRouter.js.